script.js

```diff
@@ -1,5 +1,10 @@
+<<<<<<< HEAD
 import { api } from './constants.js';
 import { ClearCache, ErrorEvent, EventKey } from './shared.js';
+=======
+import { api, logstr, DefaultOptions } from './constants.js';
+import { ClearCache } from './shared.js';
+>>>>>>> 464e512 (suspend user block collection)
```

Comment:
looks like you have a rebase conflict here

```suggestion
import { api, logstr, DefaultOptions } from './constants.js';
import { ClearCache, ErrorEvent, EventKey } from './shared.js';
```

Reply:
Ah apologies, my git had been acting up as windows forcibly turned on ASLR in a recent update.